dojo/unittests/test_nexpose_parser.py

```diff
@@ -0,0 +1,23 @@
+from django.test import TestCase
+from dojo.tools.nexpose.parser import NexposeFullXmlParser
+from dojo.models import Test
+
+
+class TestNexposeParser(TestCase):
+    def test_nexpose_parser_has_no_finding(self):
+        testfile = open("dojo/unittests/scan/nexpose/no_vuln.xml")
+        parser = NexposeFullXmlParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(0, int(parser.items))
+
+    def test_nexpose_parser_has_one_finding(self):
+        testfile = open("dojo/unittests/scan/nexpose/one_vuln.xml")
+        parser = NexposeFullXmlParser(testfile, Test())
+        testfile.close()
+        self.assertEqual(1, int(parser.items))
+
+    def test_nexpose_parser_has_many_finding(self):
+        testfile = open("dojo/unittests/scan/nexpose/many_vulns.xml")
```

Comment:
Change `dojo/unittests/scan` -> `dojo/unittests/scans` and Travis should be happy.

Reply:
Thanks a lot i will commit again